### DISCUSSION TITLE
ONTO WALLET

### DISCUSSION THREAD

**Discussion by chenyoh** [WalletConnect/walletconnect-monorepo#1413] [Developer Support]
Hi, I'm a developer at ONTO Wallet and I need to modify the configuration on Wallet Connect due to a business change, However, I do not have an account to log in Wallet Connect (because the information of creating ONTO Wallet on Wallet Connect was created by submitting PR on Github), may I ask what should I do in this case?
<img width="1251" alt="截屏2022-09-07 17 50 0026" src="https://user-images.githubusercontent.com/19606566/188856630-e5d3ec70-32d4-40f7-9bd2-d75495a2697d.png">

**Comment by looter** [comment]
hi @chenyoh can you send me your email address or send it via discord? I can go ahead and update it to an email address you have access to that you create in the cloud explorer.
